packages/web-components/src/components/masthead/masthead-container.ts

```diff
@@ -611,6 +611,8 @@ _  */
         <dds-masthead-menu-button
           button-label-active="${ifNonNull(menuButtonLabelActive)}"
           button-label-inactive="${ifNonNull(menuButtonLabelInactive)}"
+          button-label="Open menu"
```

Comment:
@RaphaelAmadeu hey Raph, we probably should have this as a field that eventually gets passed in from the navigation data (I think we have that in there, do you recall @kennylam ?
We shouldn't hard code the english label for this, maybe have those as default value if nothing is passed in.